sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features Added
 
+* Added objects used to call Azure Functions.
```

Comment:
I would mention this is for Agents. Perhaps make this clear in the other items here as well.